src/content/contributor-docs/es/access.md

```diff
@@ -0,0 +1,60 @@
+
+## Tipos de acceso
+
+Aumentar el acceso no se trata de expandir el número neto de personas en la comunidad de p5.js. Es un compromiso continuo de hacer que p5.js esté disponible y sea accesible para personas que han sido excluidas de la comunidad de p5.js como consecuencia de la opresión estructural. Este compromiso se extiende a las herramientas y plataformas que ofrece p5.js. También incluye la composición, toma de decisiones y acciones del liderazgo de p5.js. Nos resistimos a la cultura tecnológica de la velocidad, el crecimiento y la competencia. Damos prioridad a la intencionalidad, la lentitud, la adaptación y la responsabilidad como actos de cuidado colectivo.
+
+Aquí, acceso significa hacer que p5.js sea equitativo para:
+
+- Personas que hablan idiomas distintos al inglés
+- Personas negras, indígenas, de color y personas de etnias marginadas
+- Personas lesbianas, gays, bisexuales, queer, en cuestionamiento, pansexuales y asexuales
+- Personas trans, de género fluido, agénero, intersexuales y de dos espíritus, mujeres y otras personas con géneros marginados
+- Personas ciegas, s/Sordas[^2] o con dificultades auditivas, discapacitadas/con una discapacidad, neurodivergentes y con enfermedades crónicas[^3]
+- Personas con bajos ingresos o falta de acceso a capital financiero o cultural
+- Personas con poca o ninguna experiencia previa en código abierto y programación creativa
+- Personas de diversos trasfondos educativos
+- Personas de todos los grupos de edad, incluidos niños y ancianos
+- Personas con una variedad de habilidades tecnológicas, herramientas y acceso a internet
+- Personas de diversos trasfondos religiosos
+- Otras personas que son sistemáticamente excluidas e históricamente subrepresentadas
+- Y todas las intersecciones de estos grupos
+
+Reconocemos la complejidad de los términos utilizados para describir nuestras respectivas identidades. El lenguaje es matizado, evolutivo y controvertido. Esta lista no es exhaustiva. Hacemos un intento de nombrar y rendir cuentas a nuestros compromisos y a las diversas necesidades de la comunidad de p5.js.
+
+
+### Ejemplos
+
+Estos son ejemplos de esfuerzos que creemos aumentan el acceso:
+
+- Traducir documentación y otros materiales a más idiomas, descentralizando el imperialismo lingüístico[^4] (por ejemplo, [Procesamiento en idioma kuna](https://medium.com/@ProcessingOrg/culture-as-translation-processing-in-kuna-language-with-rolando-vargas-and-edinson-izquierdo-8079f14851f7) de Rolando Vargas y [Pê Cinco: Internacionalización y popularización para hablantes del portugués](https://medium.com/processing-foundation/translating-p5-js-into-portuguese-for-the-brazilian-community-14b969e77ab1) de Felipe Santos Gomes, Julia Brasil, Katherine Finn Zander y Marcela Mancino)
```

Comment:
En la frase "Processing in Kuna Language", _processing_ hace referencia al nombre de la fundación creadora de p5.js, por lo que no debe ser traducido.